src/components/_internal/sw-floating-ui/sw-floating-ui.spec.ts

```diff
@@ -56,7 +58,7 @@ describe("sw-floating-ui", () => {
     wrapper = createWrapper();
 
     const triggerSlot = wrapper.find("#triggerSlotContent");
-    expect(triggerSlot.exists()).toBe(true);
+    expect(triggerSlot.exists()).toBeTruthy();
```

Comment:
This was a default recommendation of the vitest eslint plugin. Therefore I used it here. In this case I think a truthy value also makes sense because it doesn't matter what the type of return value is. It matters that it found something